script/smoke_test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
+
+# This is an integration test to generally check the quality of parsing.
+
+SCALA_SCALA_LIBRARY_EXPECTED=87
+SCALA_SCALA_COMPILER_EXPECTED=46
+DOTTY_COMPILER_EXPECTED=35
```

Comment:
_sad trombone music_

Reply:
Only way from here is up. *optimistic trombone music*

Reply:
https://orchestraexcerpts.com/trombone-wagner-die-walkure-ride-valkyries-2-excerpts/